tests/modules/aioseop/test-All_in_One_SEO_Pack-aiospMrtGetUrl.php

```diff
@@ -105,7 +107,13 @@ public function setup_posts( $how_many = 0 ) {
 		 * @link https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#shared-setup-between-related-tests
 		 */
 		public function tearDown() {
-			// Do Stuff...
+			$this->clean();
+
+			$this->go_to( site_url() );
+			unset( $GLOBALS['wp_query'], $GLOBALS['wp_the_query'] );
+			$GLOBALS['wp_the_query'] = new WP_Query();
+			$GLOBALS['wp_query']     = $GLOBALS['wp_the_query'];
+
 			parent::tearDown();
```

Comment:
@EkoJr teardown is automatically called after every test. is this required?

Reply:
Yes. https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#shared-setup-between-related-tests